Ginger/GingerCore/ALM/JIRA/Bll/JiraExportManager.cs

```diff
@@ -281,6 +321,114 @@ public bool ExecuteDataToJira(BusinessFlow bizFlow, PublishToALMConfig publishTo
+                                    JiraZephyrStepResult currentStepResult = stepResults.Where(z => z.StepId.ToString() == act.ExternalID).FirstOrDefault();
+                                    if (currentStepResult != null)
+                                    {
+                                        currentStepResult = ((JiraManagerZephyr)jiraRepObj.TestAlmManager()).UpdateZephyrStepResult(ALMCore.DefaultAlmConfig.ALMUserName, ALMCore.DefaultAlmConfig.ALMPassword,
+                                                                                                                                     ALMCore.DefaultAlmConfig.ALMServerURL, currentStepResult.Id,
+                                                                                                                                     ((int)zephyrStepStatus)).DataResult;
+                                        string comment = CreateCommentForRun(act.Acts.ToList());
+                                    }
+                                    else
+                                    {
+                                        currentStepResult = ((JiraManagerZephyr)jiraRepObj.TestAlmManager()).CreateZephyrStepResult(ALMCore.DefaultAlmConfig.ALMUserName, ALMCore.DefaultAlmConfig.ALMPassword,
+                                                                                                                                     ALMCore.DefaultAlmConfig.ALMServerURL,
+                                                                                                                                     new JiraZephyrStepResult(  Convert.ToInt32(act.ExternalID),
+                                                                                                                                                                currentActivitiesGroupExecution.IssueId.ToString(),
+                                                                                                                                                                currentActivitiesGroupExecution.Id,
+                                                                                                                                                                ((int)zephyrStepStatus).ToString())).DataResult;
+                                        if (currentStepResult == null)
+                                        {
+                                            resultFlag = false;
+                                        }
+                                    }
+                                }
+
+                                zephyrIssueStatus = ConvertGingerStatusToZephyr(actGroup.RunStatus);
+                                currentActivitiesGroupExecution = ((JiraManagerZephyr)jiraRepObj.TestAlmManager()).UpdateExecution(ALMCore.DefaultAlmConfig.ALMUserName, ALMCore.DefaultAlmConfig.ALMPassword,
```

Comment:
Codacy found an issue: [Remove this useless assignment to local variable 'currentActivitiesGroupExecution'.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6280018)